### PR TITLE
Accomodated descriptions within v1 <-> v2 transformations

### PR DESCRIPTION
**Pull request by czardoz**

### CODE REVIEW

File: lib/converters/converter-v1-to-v2.js [hunk 1]
```diff
@@ -130,15 +154,32 @@ _.extend(Builders.prototype, {
...
-                key: key,
-                value: value,
-                description: ''
-            };
-            _.startsWith(key, headersCommentPrefix) && (thisHeaderObject.disabled = true);
-            return thisHeaderObject;
+
+        var headers = [],
+            traversed = {},
+            // deep clone is needed as we are cloning an array of objects. (2nd level nesting)
```
Comment (czardoz) [inline lib/converters/converter-v1-to-v2.js]: comment should be removed

File: lib/converters/converter-v2-to-v1.js [hunk 2]
```diff
@@ -334,6 +334,12 @@ _.extend(Builders.prototype, {
         _.map(units, function (unit) {
             request[unit] = self[unit](item);
         });
+
+        // description transformations for v2 to v1
+        request.queryParams = item.request.url.query;
```
Comment (czardoz) [inline lib/converters/converter-v2-to-v1.js]: v2 might contain URLs as strings, make sure you parse the URL to an object before doing this.

Comment (kunagpal) [inline lib/converters/converter-v2-to-v1.js]: Fixed.

**Comment by kunagpal** [comment]
TODO:
- [ ] Add unit tests for description transformations

**Pull request merged**